docker-compose.yml

```diff
@@ -171,6 +171,7 @@ services:
       nofile:
         soft: 4096
         hard: 4096
+    user: root
```

Comment:
I am a bit wary of doing this, wondering if there is a better approach here

Reply:
I think this problem of kafka image from confluence, this line likes a work around for upgrading

Reply:
have you got any idea @hubertdeng123

Reply:
I was able to get around this with something like this:

```
chown -R 1000:1000 /var/lib/docker/volumes/sentry-zookeeper/_data
chown -R 1000:1000 /var/lib/docker/volumes/sentry-self-hosted_sentry-zookeeper-log/_data
```

I also ran `chown -R 1000:1000 /var/lib/docker/volumes/sentry-kafka/_data` but I am not sure if that is needed

Reply:
Can you try that?

Reply:
Given that an approach for this may require us to write a script of some sort to properly set the permissions of docker volumes, I think it is more reasonable for us to take on the brunt of this work in https://github.com/getsentry/self-hosted/pull/2988

Reply:
do you need a test?

Reply:
and if you think your commit is better, let me know to close this pr, thank!